Sources/SymbolKit/CMakeLists.txt

```diff
@@ -0,0 +1,69 @@
+#[[
+This source file is part of the DocC open source project
+
+Copyright (c) 2024 Apple Inc. and the DocC project authors
+Licensed under Apache License v2.0 with Runtime Library Exception
+
+See https://swift.org/LICENSE.txt for license information
+#]]
+
+add_library(SymbolKit
+  Mixin/Mixin+Equals.swift
```

Comment:
I have similar concerns as I raised in https://github.com/swiftlang/swift-docc/pull/818#pullrequestreview-2442381197 about the ongoing maintenance burden of this separate file list. It is _slightly_ lessened in this repository because it doesn't get updated as frequently but I like if this was automatically kept up-to-date or at least if there was an easy way for another contributor to regenerate this file list based on `find Sources/SymbolKit -type f -name "*.swift" | sort` or something similar.

Reply:
That is effectively what the list is - but this should only be run on the CI as you may have other sources which are not yet included in the build in your local tree.